README.md

```diff
@@ -4,227 +4,194 @@
 
-The container image has additional tools to handle these dependencies for
-multiple Ruby versions. Check out the `scripts` directory if you are curious.
+This component needs access to a Redis database to perform the following actions:
+ * Enqueue reports, which will be processed by the Apisonator worker component
+ * Perform authorization of the requests
+This two actions can be configured to be performed in different Redis
+databases if desired (see the Configuration section).
 
-#### Workflow
+Finally, another API is provided to configure services in Apisonator. This API
+is named 'Internal API'. You can also generate the documentation of the
+Internal API with Rake tasks.
 
-Your project directory within the container is sync'ed with your local clone of
-the project so that changes in one reflect instantly in the other.
+For a quick Apisonator test, Random services can be created and configured in
+backend via the use of the 'Buddhi' tool located in our performance tests toolkit,
+located in [3scale perftest-toolkit](https://github.com/3scale/perftest-toolkit/).
 
-We recommend editing code and committing locally, and executing tests within the
-container, since the container won't have your own tools and configurations.
+This component may also be referred as '3scale_backend'.
 
-### On local machine (unsupported)
+### Apisonator worker
 
-This is **unsupported** and **strongly discouraged** because it is the source of a lot
-of headaches and potential problems and introduces the need for us to document
-dependencies and config files here that are very likely to be outdated and
-incomplete.
+It is responsible for performing background tasks off-loaded from
+backend-listeners (enqueued jobs).
 
-Bear in mind that we can only provide best effort support for this.
+The worker component takes care of running these enqueued jobs, mainly related
+to reporting of previous traffic.
 
-1. Install Redis v2.8.19 (or whichever version is recommended for use).
-2. Run `bundle install`.
-3. Configure Redis in `~/.3scale_backend.conf` as:
+This component needs access to a Redis database to perform the following actions:
+ * Dequeue the report jobs that have been submitted to the Redis database by the
+   Apisonator listener/s or the Apisonator failed jobs rescheduler
 
-```
-ThreeScale::Backend.configure do |config|
-  config.redis.proxy = 'localhost:6379'
-end
-```
+This component may also be referred as '3scale_backend_worker'.
 
-## Running tests
+### Apisonator failed jobs rescheduler
 
-You can either run them manually in the container-based development environment
-or have a container launched just for running the tests. For the latter you just
-need to run `make test`, and you can configure any additional environment
-variables for the test scripts like so:
+This is a simple task that acts as a cron scheduler to requeue jobs that failed
+when being processed by an Apisonator worker. The jobs are requeued into
+the Redis database.
```

Comment:
I think it is best to refer to the background jobs queue database (we can mention Resque using Redis when we define it above).

Reply:
:+1: